CONTRIBUTING.md

```diff
@@ -15,22 +15,23 @@ __jsii__, it relies on the following toolchains:
  - [Python 3.6.5](https://www.python.org/downloads/release/python-365/)
  - [Ruby 2.5.1](https://www.ruby-lang.org/en/news/2018/03/28/ruby-2-5-1-released/)
 
-Our CI/CD uses the "superchain" image from [aws-delivlib](https://github.com/awslabs/aws-delivlib). 
+A `jsii/superchain` Docker image is available and includes all the required
+tools, including those necessary to build `jsii` itself.
 
-This image can also be used locally like this (note that initial build may take quite some time):
+This image can also be built and used locally:
 
 ```console
-$ git clone git@github.com:awslabs/aws-delivlib.git
-$ cd aws-delivlib/superchain
+$ git clone git@github.com:aws/jsii.git
```

Comment:
Why clone the jsii repo again? This is already in the context of the jsii repo, isn't it?